### PR TITLE
Update sample readme & fix vulnerabiltiy

### PR DESCRIPTION
**Pull request by hermanwenhe**

### CODE REVIEW

**Review by GavinGu07 — APPROVED** [review]
LGTM

File: Word-HelloWorld-TaskPane-JS/README.md [hunk 1]
```diff
@@ -1,37 +1,65 @@
...
+- Office connected to a Microsoft 365 subscription. You might qualify for a Microsoft 365 E5 developer subscription through the [Microsoft 365 Developer Program](
+https://developer.microsoft.com/microsoft-365/dev-program), see [FAQ](
+https://learn.microsoft.com/office/developer-program/microsoft-365-developer-program-faq#who-qualifies-for-a-microsoft-365-e5-developer-subscription-) for details.
+Alternatively, you can [sign up for a 1-month free trial](
+https://www.microsoft.com/microsoft-365/try?rtc=1)
+or [purchase a Microsoft 365 plan](
+https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
 
-## Run and Debug Word Add-in
+### Run the add-in using Office Add-in Dev Kit
```
[the remark this answers is not in the document]
Comment (hermanwenhe) [reply Word-HelloWorld-TaskPane-JS/README.md:17]: Make sense. Will update.

File: Word-HelloWorld-TaskPane-JS/README.md [hunk 2]
```diff
@@ -1,11 +1,16 @@
-# Build Word add-ins using Office Add-in Dev Kit
+# Word Hello World Task Pane Add-in
+
+<img src="./assets/thumbnail.png" width="800"/>
+
+This is an Excel hello-world add-in that simply inserts "Hello World" to canvas.
```
Comment (MingjiaLiu1995) [inline Word-HelloWorld-TaskPane-JS/README.md:5]: Word hello-world

File: Word-HelloWorld-TaskPane-JS/README.md [hunk 3]
```diff
@@ -14,7 +19,10 @@ https://www.microsoft.com/microsoft-365/try?rtc=1)
 or [purchase a Microsoft 365 plan](
 https://www.microsoft.com/microsoft-365/buy/compare-all-microsoft-365-products).
 
+
 ### Run the add-in using Office Add-in Dev Kit
+[Office Add-in Dev Kit](https://marketplace.visualstudio.com/items?itemName=msoffice.microsoft-office-add-in-debugger) is an end-to end developer tool for building Office add-ins. You can use this tool to easily creating, running and debugging, and managing the lifecycle of an Office add-in.
+<br><img src="./assets/devkit_preview.png" width="800"/>
```
Comment (MingjiaLiu1995) [inline Word-HelloWorld-TaskPane-JS/README.md:25]: image is broken

**Review by GavinGu07 — APPROVED** [review]

**Pull request merged**